code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -28,13 +28,17 @@ Bonus
 	symptom_delay_max = 35
 	prefixes = list("Nasal ")
 	bodies = list("Cold")
-	threshold_desc = "<b>Stealth 4:</b> The symptom remains hidden until active."
+	var/infective = FALSE
+	threshold_desc = "<b>Stealth 4:</b> The symptom remains hidden until active.<br>\
+					  <b>Transmission 12:</b> The host may spread the disease through sneezing."
 
 /datum/symptom/sneeze/Start(datum/disease/advance/A)
 	if(!..())
 		return
 	if(A.stealth >= 4)
 		suppress_warning = TRUE
+	if(A.transmission >= 12)
+		infective = TRUE
```

Comment:
Infective should just happen by default for sneezing.

Reply:
no, it should not. infectivity generally shouldnt bypass needing a high transmission score, as it kinda ruins the need to spec transmission. the only time it should be able to is a high level symptom with low stats and obvious, avoidable effects (dermagraph, macrophage)

removing this aspect of sneezing was the first PR i ever made, and i still think its a good one

Reply:
Transmission 12 is pretty high though, perhaps it should be a random probability with the chance increased as infectivity goes up?

Reply:
transmission 12 is 1 over contact under this pr- I buffed this to 100% chance, but it's a 4-range spread, which is pretty damn good.

Sneezing already represents the fact it makes a virus more contagious with its high transmission attribute (4- the highest transmission symptom), and doesnt need to be any more obligatory a choice than that for contagion-based viruses. I don't want spreading viruses to find sneezing more attractive than pumping the stats, nor do i want to change its stats to compensate.